src/components/footer/footer.scss

```
@@ -0,0 +1,106 @@
+@import "../../components/styles/variables.scss";
+@import "../../components/styles/mixins.scss";
+
+
+.footer {
+    &-container {
+      height: 492px;
+      width: 1600px;
+      background-color: $black-color;
+      display: flex;
+      justify-content: space-between;
+    }
+    &-bikcraft {
+      text-decoration: none;
+      color: $white-color;
+      font-family: Poppins;
+      font-size: 34px;
+      font-weight: 600;
+      line-height: 72px;
+      letter-spacing: 0em;
+      text-align: left;
+      margin-top: 40px;
+      margin-left: 122px;
+    }
+    &-info-title {
+      font-family: Roboto;
+      font-size: 24px;
+      font-weight: 500;
+      line-height: 32px;
+      letter-spacing: 0.015em;
+      text-align: left;
+      margin-top: 60px;
+      margin-bottom: 32px;
+      text-transform: uppercase;
+    }
+    &-info-ul {
+      li {
+        list-style-type: none;
+        margin-left: -40px;
+      }
+      color: $light-gray-color;
+      font-family: Roboto;
+      font-size: 18px;
+      font-weight: 400;
+      line-height: 32px;
+      letter-spacing: 0.015em;
+      text-align: left;
+  
+      font-family: Roboto;
+      font-size: 18px;
+      font-weight: 500;
+      line-height: 32px;
+      letter-spacing: 0.015em;
```

```suggestion
      font-size: 18px;
      font-weight: 400;
      line-height: 32px;
      letter-spacing: 0.015em;
      text-align: left;
  
      font-family: Roboto;
      font-size: 18px;
      font-weight: 500;
      line-height: 32px;
      letter-spacing: 0.015em;
```

remove duplicated font style

---

done